src/main/content/_assets/js/header.js

```diff
@@ -28,4 +26,40 @@ $(document).ready(function() {
         }
     });
 
+    if ($('#whatsNewModal').attr('href')) {
+        $("#modal-body").load($('#whatsNewModal').attr('href'), function(response, status) {
+            if (status === "success") {
+                let version = `${$('#general_title').text().replace(/[^\d.]/g, '')}`;
+                let localStorageValue = localStorage.getItem('whatsNew');
+                localStorageValue = localStorageValue ? JSON.parse(localStorageValue) : {};
+        
+                if (version === localStorageValue['whatsNewVersion']) {
+                    if (!localStorageValue['didOpenWhatsNew']) {
+                        $('.toast').toast('show');
+                    }
+                }
+                else if (version !== localStorageValue['whatsNewVersion']) {
+                    updateLocalStorageValue(version, false);
+                    $('.toast').toast('show');
+                }
+    
+                function updateLocalStorageValue(whatsNewVersion, didOpenWhatsNew){
```

Comment:
put this on top level and make it more generic so other functions can update local storage too

have it take in a key and value as parameters.